### PR TITLE
Add the Notification class.

### PR DESCRIPTION
**Pull request by Hossain-Ahmed1**
added static class Notification that add string to a list of string and return a String array

### CODE REVIEW

File: src/FOHInterface/FOHController.java [hunk 1]
```diff
@@ -36,19 +37,26 @@ public int getAmountBooked(long timestamp) {
...
         Bill.setServiceChargePercentage(newCharge);
+        String notification = "The service charge now is: " + newCharge +"%";
+        Notification.addNotification(notification);
     }
 
     @Override
     public void markOrderComplete(int orderID) {
         OrderCollection.get(orderID).setComplete(true);
         OrderDAO.setOrderStatusToReady(orderID);
+        String notification = "The Order: " +  OrderCollection.get(orderID).getOrderId() + " is ready to collect.";
```
Comment (StevenAtUni) [inline src/FOHInterface/FOHController.java:50]: If you've already got the orderID, why are you getting the orderId again?

File: src/FOHInterface/FOHController.java [hunk 2]
```diff
@@ -36,19 +37,26 @@ public int getAmountBooked(long timestamp) {
...
+        Notification.addNotification(notification);
     }
 
     @Override
     public void markItemUnavailable(int itemID) {
         DishDAO.setDishUnavailable(itemID);
         MenuItemCollection.get(itemID).setAvailable(false);
+        String notification = "The dish: " +  MenuItemCollection.get(itemID).getName() + " is unavailable.";
```
Comment (StevenAtUni) [inline src/FOHInterface/FOHController.java:58]: This is good, but I'd say send the name as well as the itemID

**Review by StevenAtUni — CHANGES_REQUESTED** [review]
Almost there! Just 2 minor changes

File: src/FOHInterface/FOHController.java (hunk 1 above)
Comment (Hossain-Ahmed1) [reply src/FOHInterface/FOHController.java:50]: brains busted ill fix that

File: src/FOHInterface/FOHController.java (hunk 2 above)
Comment (Hossain-Ahmed1) [reply src/FOHInterface/FOHController.java:58]: sure

**Review by StevenAtUni — COMMENTED** [review]
Good stuff

**Pull request closed without merging**